client/redux/cart/reducers.js

```diff
@@ -0,0 +1,74 @@
+import {
+  TOGGLE_CART,
+  FETCH_CART_PRODUCTS_FAILURE,
+  FETCH_CART_PRODUCTS_SUCCESS,
+  FETCH_CART_PRODUCTS_REQUEST,
+  UPDATE_CART,
+  UPDATE_CART_ID,
+} from './constants'
+
+const checkCookies = () => {
+  const arr = document.cookie.split('; ')
```

Comment:
@AStashevskaya Lets follow `DRY` principle, since this code for cookies operatins is already written in `actions`, lets move it to some `utils/cookie.js` file and import methods for `cookie` manipulation